Mage.Sets/src/mage/cards/r/RageExtractor.java

```diff
@@ -23,7 +23,7 @@
  */
 public final class RageExtractor extends CardImpl {
 
-    private static final FilterSpell filter = new FilterSpell("a spell with {P} in its mana cost");
+    private static final FilterSpell filter = new FilterSpell("a spell with {H} (<i>a phyrexian mana symbol</i>) in its mana cost");
```

Comment:
I see the merit in this, but I'm not convinced we should be adding reminder text that doesn't exist. At least put the parentheses inside the italics

Reply:
Maybe i'll take it out--not worth tracking so closely for one card i suppose